test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md

```diff
@@ -0,0 +1,19 @@
+---
+estimate: 15m
+tags:
```

Comment:
should be `targets`, not `tags` I think

Reply:
Yeah, you're correct.  Fixed it with new commit.